cmd/printdep/printdep.go

```diff
@@ -9,18 +9,38 @@ package main
 import (
 	"flag"
 	"fmt"
+	"log"
+	"runtime"
 	"strings"
 
 	ts "tailscale.com"
 )
 
 var (
-	goToolchain = flag.Bool("go", false, "print the supported Go toolchain git hash (a github.com/tailscale/go commit)")
+	goToolchain    = flag.Bool("go", false, "print the supported Go toolchain git hash (a github.com/tailscale/go commit)")
+	goToolchainURL = flag.Bool("go-url", false, "print the URL to the tarball of the Tailscale Go toolchain")
 )
 
 func main() {
 	flag.Parse()
 	if *goToolchain {
 		fmt.Println(strings.TrimSpace(ts.GoToolchainRev))
 	}
+	if *goToolchainURL {
+		var suffix string
+		switch runtime.GOARCH {
+		case "amd64":
+			// None
+		case "arm64":
```

Comment:
We build static binaries for a number of other architectures like MIPS, will we need any handling here?

Reply:
We don't; only four: https://github.com/tailscale/go/releases

Reply:
(That is, these are the four platforms we support for development)